src/rosdep2/lookup.py

```diff
@@ -433,13 +438,13 @@ def resolve_all(self, resources, installer_context, implicit=False):
 
         return resolutions_flat, errors
 
-    def resolve(self, rosdep_key, resource_name, installer_context):
+    def resolve(self, rosdep, resource_name, installer_context):
```

Comment:
Ditto.

Reply:
this is not new argument, just to rename variable name, since we have changed from list of keys to `catkin_pkg.package.Dependency` object
``
        :param rosdep_key: rosdep key to resolve
        :param rosdeps: rosdep key (catkin_pkg.package.Dependency object) to resolve
``

Reply:
Yep understood.